app/Migrations/Version20200804090405.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace CultuurNet\UDB3\Silex\Migrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+class Version20200804090405 extends AbstractMigration
+{
+    public function up(Schema $schema): void
+    {
+        $table =  $schema->getTable('productions');
+
+        // Increase the max-length of the name column from 32 to 255.
```

Comment:
Oh wow, I didn't realise 255 wasn't the default 😮

Reply:
Me neither, I only noticed it when opening the table in Sequel Pro to check that the production I `POST`ed was actually inserted.

Reply:
Good spot then 👍